### PR TITLE
Fix stdlib paths in stacktraces

### PR DESCRIPTION
**Pull request by ghost**
this hopefully solves #35191

### CODE REVIEW

File: base/errorshow.jl [hunk 1]
```diff
@@ -496,7 +496,8 @@ function show_method_candidates(io::IO, ex::MethodError, @nospecialize kwargs=()
                     print(iob, "; ")
                     join(iob, kwords, ", ")
                 end
-                print(iob, ")")
+		file ,line =updated_methodloc(method)
```
Comment (KristofferC) [inline base/errorshow.jl:499]: indentation and spaces look a bit off here

Comment (sagnik-chatterjee) [reply base/errorshow.jl:499]: Ok , i'll rectify.

**Comment by oschulz** [comment]
Is this still alive? I'd love to see this fixed (thanks, @sagnik-chatterjee)

**Comment by KristofferC** [comment]
I realize now that the approach here is not correct (sorry for giving a bad suggestion). It will just give the line number to the start of the method.

**Comment by musm** [comment]
closed in favor of https://github.com/JuliaLang/julia/pull/37427

**Pull request closed without merging**